StudentProject/StudentDetails.java

```diff
@@ -48,88 +27,86 @@ public static void main(String[] args) {
             ch = s.nextInt();
 
             switch (ch) {
-                case 1:
+                case ADD_STUDENT:
                     System.out.println("--------------------------------------------------");
                     System.out.println("Please provide the data for the student to store. ");
                     System.out.print("Student id: ");
-                    int sno = s.nextInt();
+                    int eno = s.nextInt();
                     System.out.print("First name: ");
-                    String sfname = s1.nextLine();
+                    String StudentFirstName = s1.nextLine();
                     System.out.print("Last name: ");
-                    String slname = s1.nextLine();
-
-                    c.add(new StudentData(sno, sfname, slname));
+                    String StudentLastName = s1.nextLine();
+                    c.add(new StudentData(eno, StudentFirstName, StudentLastName));
                     break;
-                case 2:
+                case SHOW_STUDENT:
                     System.out.println("--------------------------------------------------");
-                    System.out.println("Id fname lname");
+                    System.out.printf("%-10s %-15s %-15s", "id", "firstname", "lastname");
+                    System.out.println();
                     Iterator<StudentData> i = c.iterator();
                     while (i.hasNext()) {
                         StudentData e = i.next();
-                        System.out.println(e);
+                        System.out.printf("%-10s %-15s %-15s", e.getStudentID(), e.getStudentFirstName(), e.getStudentLastName());
+                        System.out.println();
                     }
-                    System.out.println("--------------------------------------------------");
+                    System.out.println("----------------------------");
                     break;
-                case 3:
+                case SEARCH_STUDENT:
                     boolean found = false;
                     System.out.println("Please give the student id to search? ");
                     System.out.print("Student id:");
-                    int sno1 = s.nextInt();
+                    int StudentId = s.nextInt();
                     System.out.println("--------------------------------------------------");
                     i = c.iterator();
```

Comment:
Instead of an Iterator, you can also use a for each loop. for(StudentData currentStudent : c) { ... } or even easier, use streams and filter (https://rit-moodle.inso.tuwien.ac.at/mod/page/view.php?id=662).